examples/conways_game_of_life_fast/app.py

```diff
@@ -20,11 +20,15 @@
     },
 }
 
+gol = GameOfLifeModel(10, 10)
+
+AlivePlot = make_plot_measure("Cells alive", "Fraction alive")
```

Comment:
Does this plot nicely on a single plot? Since `"Cells alive"` might be on the 0-100 scale, while `"Fraction alive"` is in the 0-1 scale.

(they are a bit duplicate though, so removing one is also fine)

Reply:
not really, I split this into 2 plots